src/FsAutoComplete.Core/Commands.fs

```diff
@@ -238,6 +238,33 @@ type Commands (serialize : Serializer, backgroundServiceEnabled) =
         let loader = Dotnet.ProjInfo.Workspace.Loader.Create(config)
         loader, checker.CreateFCSBinder(NETFrameworkInfoProvider.netFWInfo, loader)
 
+    let pageForLint (code: string) =
+        let codeNumber = code.Substring (2) |> int
+        match codeNumber with
+        | 1 | 2 | 3 -> Some "http://fsprojects.github.io/FSharpLint/TupleFormatting.html"
+        | 4 | 5 | 6 | 7 -> Some "http://fsprojects.github.io/FSharpLint/PatternMatchFormatting.html"
+        | 8 | 9 -> Some "http://fsprojects.github.io/FSharpLint/Spacing.html"
+        | 10 (* | 11 (not present?) *) | 12 -> Some "http://fsprojects.github.io/FSharpLint/Formatting.html"
+        | 13 | 14 | 15 -> Some "http://fsprojects.github.io/FSharpLint/Conventions.html"
+        | 22 | 23 | 24 | 25 | 26 | 27 | 28 | 29 | 30 | 31 | 32 | 33 -> Some "http://fsprojects.github.io/FSharpLint/SourceLength.html"
+        | 34 | 35 -> Some "http://fsprojects.github.io/FSharpLint/FunctionReimplementation.html"
+        | 36 | 37 | 38 | 39 | 40 | 41 | 42 | 43 | 44 | 45 | 46 | 47 | 48 | 49 | 50 -> Some "http://fsprojects.github.io/FSharpLint/NameConventions.html"
+        | 51 | 52 | 53 | 54 -> Some "http://fsprojects.github.io/FSharpLint/NumberOfItems.html"
+        | 55 | 56 | 57 | 58 -> Some "http://fsprojects.github.io/FSharpLint/Binding.html"
+        | 60 | 61 | 62 | 63 | 64 -> Some "http://fsprojects.github.io/FSharpLint/Typography.html"
+        | _ -> None
+
+
+    /// lint warnings come to us with the lint identifier as part of the info message,
+    /// so we parse and split that out here for processing later.
+    /// In addition we add the url to the matching help page for fsharplint
+    let enrichLintWarning (w: LintWarning.Warning) =
+        match w.Info.IndexOf ":" with
+        | -1 -> "unknown", None, w
+        | n ->
+            let code = w.Info.[0..n-1]
+            code, pageForLint code, { w with Info = w.Info.Substring(n+1).Trim() }
```

Comment:
as suggestion, if you want, can you split this code in:
- a function to split a string up to char (`string -> char -> (string * string) option`) in `Utils.fs` 
- the logic itself using that function

can be a nice utility